python/ray/autoscaler/v2/tests/test_e2e.py

```diff
@@ -20,6 +20,73 @@ def is_head_node_from_resource_usage(usage: Dict[str, float]) -> bool:
     return False
 
 
+def test_autoscaler_no_churn():
```

Comment:
nice - does this test fail on the current master?

Reply:
yes after adding lag in node_provider in fake_multi_node (relevant changes for this comment now in here: https://github.com/ray-project/ray/pull/40369)